components/library/plib/Makefile

```diff
@@ -0,0 +1,34 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2021 Jake Goerzen
+
+BUILD_BITS=			32_and_64
+
```

Comment:
Do you really need 32 Bit? We want to get rid of 32 bit applications and I am reluctant to merge new packages for 32 bit.

Reply:
As far as I know 32 Bit is not need.  I'll switch to build with BUILD_BITS= 64

Reply:
Then you could simply remove the `BUILD_BITS=` line because 64 bit is default now.